include/grpc++/server_builder.h

```diff
@@ -55,30 +55,93 @@ class ServerBuilder {
   ServerBuilder();
   ~ServerBuilder();
 
-  /// Options for synchronous servers.
-  enum SyncServerOption {
-    NUM_CQS,         ///< Number of completion queues.
-    MIN_POLLERS,     ///< Minimum number of polling threads.
-    MAX_POLLERS,     ///< Maximum number of polling threads.
-    CQ_TIMEOUT_MSEC  ///< Completion queue timeout in milliseconds.
-  };
+  //////////////////////////////////////////////////////////////////////////////
+  // Primary API's
+
+  /// Return a running server which is ready for processing calls.
+  /// Before calling, one typically needs to ensure that:
+  ///  1. a service is registered - so that the server knows what to serve
+  ///     (via RegisterService, or RegisterAsyncGenericService)
```

Comment:
If the goal here is to optimize the learning path for the common case, would it be better to avoid mention of `RegisterAsyncGenericService`, since most folks won't need or use that?

Reply:
I might be weird but...

Given the 'right' RegisterService is on screen and syntax highlighted just below here, my eye tends to see it pretty quickly.

For that reason, I think it's worth calling out that there are variants here since this starts to act like a TODO list before calling this method.